### DISCUSSION TITLE
Application ChatsApp ne fonctionne plus

### DISCUSSION THREAD

**Discussion by AnthonyDavid3110** [web-classroom/web-classroom.github.io#21] [Q&A : Pratique / Labos] [answered]
Bonjour,
Impossible de me connecter à l'application pour le laboratoire : 
![image](https://github.com/web-classroom/web-classroom.github.io/assets/70233189/76360206-2d06-4428-b5e0-01958fea5e07)
Je ne crois pas avoir d'autre moyen pour récupérer les flags.

**Comment by OlivierLmr** [comment, marked as answer]
En effet, désolé pour ça. C'est normalement réglé maintenant.

**Reply by AnthonyDavid3110** [reply]
C'est tout bon merci !
